docs/react-wiki-archive/@uifabric-migration.md

```diff
@@ -0,0 +1,121 @@
+The `migration` package provides code modifications and notes in order to ease upgrades between fabric versions. **This package is only applicable when upgrading from `office-ui-fabric-react` 6 to 7.**
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31202&commitId=55725522ff3910154ae604982d203ef25d6cd0aa&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Root - RTL.default.chromium.png|0| Added|
|ActivityItem.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>